blockwatch/watcher.go

```diff
@@ -166,7 +166,7 @@ func (w *Watcher) pollNextBlock() error {
 		if err == ethereum.NotFound {
 			log.WithFields(log.Fields{
 				"blockNumber": nextBlockNumber,
-			}).Info("block header not found")
+			}).Trace("block header not found")
```

Comment:
@fabioberger This log was getting pretty noisy. I think it is always expected to happen while we are waiting for the next block to be mined. Are you okay with changing this to `Trace`?

Reply:
Yeah totally, thanks for making that change. It is an expected condition.